runtime/zlib/trees.c

```diff
@@ -1144,7 +1144,7 @@ local int detect_data_type(s)
         if (s->dyn_ltree[n].Freq != 0)
             return Z_TEXT;
 
-    /* There are no "black-listed" or "white-listed" bytes:
+    /* There are no "block-listed" or "white-listed" bytes:
```

Comment:
I think the search / replace missed "white-listed" as I see a couple of occurrences of it in this file.

Can you put the zlib changes in a separate PR?  I want to think through the implications of modifying source that's maintained elsewhere and the impacts on our ability to consume it in the future. 

Ideally, changes like this would be done upstream in the zlib project first and then consumed from there to OpenJ9

Reply:
zlib is maintained in https://github.com/madler/zlib We should try to get these changes accepted upstream first.  If they won't accept them, we can look at maintaining this patch indefinitely

Reply:
Okay. I will create an issue there.

Reply:
I've removed the zlib changes from this pr.